engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -179,7 +180,7 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef, val wor
 
   private def validateBackendAssignments(calls: Seq[Call], workflowOptions: WorkflowOptions, defaultBackendName: Option[String]): ErrorOr[Map[Call, String]] = {
     val callToBackendMap = Try {
-      calls map { call =>
+      ListMap(calls map { call =>
```

Comment:
Could you rework this so you're not passing a 10 line expression into the constructor of ListMap?